### ISSUE TITLE
typo when talking to Umar in Fallhaven

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#316]
```
i tried to see if this was already reported but i didn't find anything.

when you are talking to Umar in the thieves guild and he is telling you about 
the conflict between the guards and priests, he says that the following:

"Also, the rumor is that the priests of the Shadow are still doing their 
rituals, despite of them being banned."

it should read something to the effect of "...despite most of them being 
banned..."
```

Original issue reported on code.google.com by `blue...@gmail.com` on 19 Aug 2012 at 6:15

**Comment by GoogleCodeExporter** [comment]
```
Updated in b01b16faf4279cc3c1285b9d81d3f636aba50636 (not pushed yet). Thanks 
bluers0!
The new text is now "Also, the rumor is that the priests of the Shadow are 
still doing their rituals, despite the fact that most of the rituals have been 
banned."
```

Original comment by `oskar.wi...@gmail.com` on 13 Sep 2012 at 5:12
- Changed state: **Ready**
- Added labels: **Milestone-0.6.12**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.12.
```

Original comment by `oskar.wi...@gmail.com` on 28 Oct 2012 at 7:44
- Changed state: **Fixed**
